h/templates/emails/reply_notification.txt.jinja2

```diff
@@ -1,15 +1,11 @@
-{{ reply_user.username }} has just left a reply to your annotation on "{{ document_title }}".
-{{ document_url }}
+{{ reply_user.username }} has left a reply to your annotation {{ reply_url }} on "{{ document_title }}":
```

Comment:
I'm not sure having `{{ reply_url }}` in this line in the plain text version makes much sense.

> seanh has left a reply to your annotation https://hpt.is/pHdz_jxhEeakRBv01oK7Sg/example.com/foo/bar on "Example Page":

This looks odd, I think.